base/src/com/google/idea/blaze/base/ideinfo/RustIdeInfo.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2022 The Bazel Authors. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.idea.blaze.base.ideinfo;
+
+import com.google.common.collect.ImmutableList;
+import com.google.devtools.intellij.ideinfo.IntellijIdeInfo;
+
+import javax.annotation.Nullable;
+import java.util.Objects;
+
+/** Ide info specific to Rust rules. */
+public final class RustIdeInfo implements ProtoWrapper<IntellijIdeInfo.RustIdeInfo> {
```

Comment:
I've added `RustIdeInfo` because every other language has one, but I don't know if it's being used for anything.